cmd/utils/flags.go

```diff
@@ -1531,6 +1531,8 @@ func MakeGenesis() *core.Genesis {
 		genesis = core.DefaultLocalGenesisBlock(consensusEngine, genesisNonce)
 	case params.DevName:
 		Fatalf("Developer chains are ephemeral")
+	default:
+		genesis = core.DefaultGenesisBlock(genesisNonce)
```

Comment:
Any reason why this needs to be a valid path? I'd prefer to fail early for someone who might have launched with an empty flag. Seems redundant with dev and local to have a third "default".

Reply:
Without this, it is not possible to construct a default node, without setting up Viper. That's not great in general (we should always be able to construct a default with no configuration), but its particularly difficult in the tests

Reply:
Understood as far as the need for default for use in testing. Now, why don't we use local or dev as the default? I think having 3 non canonical/test networks (Local, Dev, and "Default") is confusing.

Reply:
I didn't invent that. `DefaultGenesisBlock()` already exists for the purpose of getting the default value. I agree having so many "canonical" networks is confusing, but that's a whole other discussion, lol

"default" should be mainnet, but sometime ago Karl had us strip out references to mainnet, so now it appears even more awkward :shrug:

Reply:
That is fair you're indeed not creating a new genesis block, however can we not add/introduce it as the default case? I think adding it here now will make it harder to remove later if/when we consolidate.

Out of the 3 there's a better case for keeping dev or especially local than a generic "default." So we could make one of those the fallback.

Reply:
I disagree. Generic default or mainnet is what we want. We will not remove this, but we might rename it to 'MainnetGenesis() ' or something